src/app/component/header/header.component.html

```diff
@@ -1,24 +1,17 @@
 <nav
-  class="navbar fixed-top navbar-expand-sm navbar-light bg-light"
-  style="padding: 0px"
+  class="navbar fixed-top navbar-expand-sm navbar-light bg-light p-0"
 >
   <div class="container-fluid">
-    <a
-      href="#"
-      class="navbar-brand h1"
-      routerLink="/dashboard"
-      style="margin: 0px"
-    >
+    <a class="navbar-brand" routerLink="/dashboard">
       <img
-        class="d-inline-block"
+        class="nav-img d-inline-block"
         src="assets/img/logo1.png"
-        width="35"
-        height="35"
+        alt="Jyoti Techn"
       />
       <span>APPLICATION</span></a
     >
 
-    <div class="navigation">
+    <div class="navigation" *ngIf="userName">
       <div class="container-fluid">
         <div class="collapse navbar-collapse">
           <img src="assets/img/logo1.png" alt="" class="img-responsive" />
```

Comment:
on line 23 style is used for color please move that into style file

Reply:
I have Remove 23 line color style

Reply:
I have change inline style